init/main.c

```diff
@@ -8,9 +8,7 @@
  *  Moan early if gcc is old, avoiding bogus kernels - Paul Gortmaker, May '96
```

Comment:
Remove the comment as it takes up unnecessary room

Reply:
You forgot to define every datatype. you could do this:

#define c = char 

etc.

This way, you can optimize EVEN MORE!